common/dex/base/config-map.yaml

```diff
@@ -27,6 +27,6 @@ data:
     staticClients:
     # https://github.com/dexidp/dex/pull/1664
     - idEnv: OIDC_CLIENT_ID
-      redirectURIs: ["/authservice/oidc/callback"]
+      redirectURIs: ["http://localhost:8080/oauth2/callback"]
```

Comment:
Can we actually:
1. Have an `oauth2-proxy` overlay, that sets this configmap
2. Use `/oauth2/callback` instead of hardcoding the protocol and host

I want us for this release to expect dex and oidc-authservice to work out of the box, but still give the option to users to easily use oauth2-proxy if they want